src/tests/ftest/control/daos_server_helper.py

```diff
@@ -56,31 +64,30 @@ def test_daos_server_helper_format(self):
         # Prep server for format, run command under non-root user
         # Note: This will just report the presence of PMem namespaces if the NVDIMMs are already
         #       configured in AppDirect interleaved mode and namespaces have been created.
-        self.log.info("(1)Performing NVMe storage prepare")
+        self.log_step("Perform NVMe storage prepare as non-root")
         try:
             self.server_managers[0].prepare_storage(user, False, True)
         except ServerFailed as err:
-            self.fail("##(1)Failed preparing NVMe as user {}: {}".format(user, err))
+            self.fail(f"Failed to prepare NVMe as user {user}: {err}")
 
         # Start server
-        self.log.info("(2)Starting server as non-root")
+        self.log_step("Start server as non-root")
         try:
             self.server_managers[0].detect_format_ready()
         except ServerFailed as error:
-            self.fail(
-                "##(2)Failed starting server before format as non-root user: {}".format(error))
+            self.fail(f"Failed to start server before format as non-root user: {error}")
```

Comment:
Since we've successfully started the servers via `self.server_managers[0].detect_format_ready()` and didn't use `self.start_servers()` we should manually register the tearDown step to stop the servers:
```suggestion
            self.fail(f"Failed to start server before format as non-root user: {error}")
        self.register_cleanup(self.stop_servers)
```

Reply:
Done